### ISSUE TITLE
console crashes

### ISSUE THREAD

**Issue by siddhartha-gadgil** [lihaoyi/mill#131]
I have a fork of a small project with a [mill branch(https://github.com/siddhartha-gadgil/Superficial/tree/mill). When I try to open a console the process crashes, for example
```
@ superficial.console()() 
[28/28] superficial.console 
java.lang.Exception: 1 targets failed
superficial.console
java.lang.NullPointerException
    ammonite.ops.Shellout$.executeInteractive(Shellout.scala:23)
    ammonite.ops.Shellout$.$anonfun$$percent$1(Shellout.scala:14)
    ammonite.ops.Shellout$.$anonfun$$percent$1$adapted(Shellout.scala:14)
    ammonite.ops.Command.applyDynamic(Shellout.scala:118)
    mill.modules.Jvm$.interactiveSubprocess(Jvm.scala:37)
    mill.scalalib.ScalaModule.$anonfun$console$2(ScalaModule.scala:246)
    scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)

  mill.main.ReplApplyHandler.apply(ReplApplyHandler.scala:119)
  mill.main.ReplApplyHandler.apply(ReplApplyHandler.scala:88)
  mill.define.Applicative$Applyable.apply(Applicative.scala:26)
  mill.define.Applicative$Applyable.apply$(Applicative.scala:26)
  mill.define.Task.apply(Task.scala:21)
  ammonite.$sess.cmd0$.<init>(cmd0.sc:1)
  ammonite.$sess.cmd0$.<clinit>(cmd0.sc)
```

**Comment by siddhartha-gadgil** [comment]
My guess is that the line 14 in https://github.com/lihaoyi/Ammonite/blob/ba4eb45dbc206ecd116c9ea98134bf79eb597b5e/ops/src/main/scala/ammonite/ops/Shellout.scala should be a `lazy val` - I am sorry I do not have a setup to test this quickly (a slow network means setting up the setup will take some time).

**Comment by lihaoyi** [comment]
Should be fixed by https://github.com/lihaoyi/mill/commit/65779197026a2293407afe7066ff2eaa344f157e, which will publish itself once it finishes CI

**Comment by lihaoyi** [comment]
Version https://github.com/lihaoyi/mill/releases/download/unstable/0.1.0-4-d5e8ca should fix this

**Comment by siddhartha-gadgil** [comment]
Unfortunately I get the same error, even after deleting mill, installing from the above link, cleaning `out` and re-running.

**Comment by lihaoyi** [comment]
could you try https://github.com/lihaoyi/mill/releases/download/unstable/0.1.0-5-fc2574

**Comment by siddhartha-gadgil** [comment]
Thanks. This works (gave a scala console not an ammonite REPL as I
expected).

**Comment by siddhartha-gadgil** [comment]
I tried with another project (in the same repo) and got a different error.

```
gadgil@gadgil-Vostro-470:~/code/Superficial$ mill freegroups.console
Compiling (synthetic)/ammonite/predef/DefaultPredef.sc
Compiling /home/gadgil/code/Superficial/build.sc
[28/28] freegroups.console 
Welcome to Scala 2.12.4 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_101).
Type in expressions for evaluation. Or try :help.

scala> java.lang.IncompatibleClassChangeError: Implementing class
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:467)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:73)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:368)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:362)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:361)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:467)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:73)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:368)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:362)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:361)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:467)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:73)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:368)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:362)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:361)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at scala.tools.nsc.interpreter.IMain.newCompiler(IMain.scala:240)
	at scala.tools.nsc.interpreter.IMain.<init>(IMain.scala:90)
	at scala.tools.nsc.interpreter.ILoop$ILoopInterpreter.<init>(ILoop.scala:109)
	at scala.tools.nsc.interpreter.ILoop.createInterpreter(ILoop.scala:121)
	at scala.tools.nsc.interpreter.ILoop.$anonfun$process$15(ILoop.scala:1027)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at scala.tools.nsc.interpreter.ILoop.savingReader(ILoop.scala:97)
	at scala.tools.nsc.interpreter.ILoop.$anonfun$process$11(ILoop.scala:1027)
	at scala.tools.nsc.interpreter.ILoop.startup$1(ILoop.scala:1011)
	at scala.tools.nsc.interpreter.ILoop.process(ILoop.scala:1049)
	at scala.tools.nsc.MainGenericRunner.runTarget$1(MainGenericRunner.scala:82)
	at scala.tools.nsc.MainGenericRunner.run$1(MainGenericRunner.scala:85)
	at scala.tools.nsc.MainGenericRunner.process(MainGenericRunner.scala:96)
	at scala.tools.nsc.MainGenericRunner$.main(MainGenericRunner.scala:101)
	at scala.tools.nsc.MainGenericRunner.main(MainGenericRunner.scala)
1 targets failed
freegroups.console ammonite.ops.InteractiveShelloutException
    ammonite.ops.Shellout$.executeInteractive(Shellout.scala:56)
    ammonite.ops.Shellout$.$anonfun$$percent$1(Shellout.scala:14)
    ammonite.ops.Shellout$.$anonfun$$percent$1$adapted(Shellout.scala:14)
    ammonite.ops.Command.applyDynamic(Shellout.scala:118)
    mill.modules.Jvm$.interactiveSubprocess(Jvm.scala:37)
    mill.scalalib.ScalaModule.$anonfun$console$2(ScalaModule.scala:250)
    scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
```

**Comment by lihaoyi** [comment]
Could you try `mill freegroups.repl`

**Comment by siddhartha-gadgil** [comment]
Gives an error:
```
gadgil@gadgil-Vostro-470:~/code/Superficial$ mill freegroups.repl
Cannot resolve freegroups.repl. Try `mill resolve freegroups._` to see what's available.
```

**Comment by lihaoyi** [comment]
How about on https://github.com/lihaoyi/mill/releases/download/unstable/0.1.0-7-addfbd (I just added it, sorry)

**Comment by siddhartha-gadgil** [comment]
Unfortunately `repl` gives a similar error (it is recognized as a command) as below. The main difference between the two projects is a dependency on Monix 3.0.0-M3
```
gadgil@gadgil-Vostro-470:~/code/Superficial$ mill freegroups.repl
[29/29] freegroups.repl 
Loading...
Compiling (synthetic)/ammonite/predef/interpBridge.sc
java.lang.IncompatibleClassChangeError: Implementing class
  java.lang.ClassLoader.defineClass1(Native Method)
  java.lang.ClassLoader.defineClass(ClassLoader.java:763)
  java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
  java.net.URLClassLoader.defineClass(URLClassLoader.java:467)
  java.net.URLClassLoader.access$100(URLClassLoader.java:73)
  java.net.URLClassLoader$1.run(URLClassLoader.java:368)
  java.net.URLClassLoader$1.run(URLClassLoader.java:362)
  java.security.AccessController.doPrivileged(Native Method)
  java.net.URLClassLoader.findClass(URLClassLoader.java:361)
  java.lang.ClassLoader.loadClass(ClassLoader.java:424)
  sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
  java.lang.ClassLoader.loadClass(ClassLoader.java:357)
  java.lang.ClassLoader.defineClass1(Native Method)
  java.lang.ClassLoader.defineClass(ClassLoader.java:763)
  java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
  java.net.URLClassLoader.defineClass(URLClassLoader.java:467)
  java.net.URLClassLoader.access$100(URLClassLoader.java:73)
  java.net.URLClassLoader$1.run(URLClassLoader.java:368)
  java.net.URLClassLoader$1.run(URLClassLoader.java:362)
  java.security.AccessController.doPrivileged(Native Method)
  java.net.URLClassLoader.findClass(URLClassLoader.java:361)
  java.lang.ClassLoader.loadClass(ClassLoader.java:424)
  sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
  java.lang.ClassLoader.loadClass(ClassLoader.java:357)
  java.lang.ClassLoader.defineClass1(Native Method)
  java.lang.ClassLoader.defineClass(ClassLoader.java:763)
  java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
  java.net.URLClassLoader.defineClass(URLClassLoader.java:467)
  java.net.URLClassLoader.access$100(URLClassLoader.java:73)
  java.net.URLClassLoader$1.run(URLClassLoader.java:368)
  java.net.URLClassLoader$1.run(URLClassLoader.java:362)
  java.security.AccessController.doPrivileged(Native Method)
  java.net.URLClassLoader.findClass(URLClassLoader.java:361)
  java.lang.ClassLoader.loadClass(ClassLoader.java:424)
  sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
  java.lang.ClassLoader.loadClass(ClassLoader.java:357)
  ammonite.interp.GlobalInitCompat$.initGlobal(GlobalInitCompat.scala:38)
  ammonite.interp.Compiler$$anon$2.<init>(Compiler.scala:229)
  ammonite.interp.Compiler$.apply(Compiler.scala:151)
  ammonite.interp.CompilerLifecycleManager.init(CompilerLifecycleManager.scala:80)
  ammonite.interp.CompilerLifecycleManager.preprocess(CompilerLifecycleManager.scala:49)
  ammonite.interp.Interpreter.compileRunBlock$1(Interpreter.scala:458)
  ammonite.interp.Interpreter.$anonfun$processAllScriptBlocks$9(Interpreter.scala:495)
  scala.Option$WithFilter.map(Option.scala:146)
  ammonite.interp.Interpreter.loop$1(Interpreter.scala:482)
  ammonite.interp.Interpreter.processAllScriptBlocks(Interpreter.scala:546)
  ammonite.interp.Interpreter.$anonfun$processModule$6(Interpreter.scala:346)
  ammonite.util.Catching.flatMap(Res.scala:114)
  ammonite.interp.Interpreter.$anonfun$processModule$5(Interpreter.scala:337)
  ammonite.util.Res$Success.flatMap(Res.scala:61)
  ammonite.interp.Interpreter.processModule(Interpreter.scala:329)
  ammonite.interp.Interpreter.$anonfun$initializePredef$1(Interpreter.scala:104)
  ammonite.interp.Interpreter.$anonfun$initializePredef$1$adapted(Interpreter.scala:104)
  ammonite.interp.PredefInitialization$.$anonfun$apply$7(PredefInitialization.scala:65)
  ammonite.util.Res$.$anonfun$fold$1(Res.scala:31)
  scala.collection.LinearSeqOptimized.foldLeft(LinearSeqOptimized.scala:122)
  scala.collection.LinearSeqOptimized.foldLeft$(LinearSeqOptimized.scala:118)
  scala.collection.immutable.List.foldLeft(List.scala:86)
  ammonite.util.Res$.fold(Res.scala:29)
  ammonite.interp.PredefInitialization$.apply(PredefInitialization.scala:53)
  ammonite.interp.Interpreter.initializePredef(Interpreter.scala:106)
  ammonite.repl.Repl.initializePredef(Repl.scala:107)
  ammonite.Main.run(Main.scala:181)
  ammonite.MainRunner.$anonfun$runRepl$1(Main.scala:363)
  ammonite.MainRunner.watchLoop(Main.scala:344)
  ammonite.MainRunner.runRepl(Main.scala:363)
  ammonite.Main$.main0(Main.scala:280)
  ammonite.Main$.main(Main.scala:244)
  ammonite.Main.main(Main.scala)
```

**Comment by lihaoyi** [comment]
Could you try 
```bash
mill describe freegroups.repl
```

and then run `mill show` on whatever the values of it's dependencies are

**Comment by siddhartha-gadgil** [comment]
There are two dependencies:

```
gadgil@gadgil-Vostro-470:~/code/Superficial$ mill describe freegroups.repl
[1/1] describe 
freegroups.repl(ScalaModule.scala:257)
Inputs:
    freegroups.runClasspath
    freegroups.ammoniteReplClasspath
```

When I rull `mill show` I get
```
gadgil@gadgil-Vostro-470:~/code/Superficial$ mill show freegroups.runClasspath
[1/1] show 
[11/27] freegroups.resources 
[
    {
        "path": "/home/gadgil/code/Superficial/out/freegroups/compile/dest/classes"
    },
    {
        "path": "/home/gadgil/code/Superficial/freegroups/resources"
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/typelevel/cats-kernel_2.12/1.0.1/cats-kernel_2.12-1.0.1.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/io/monix/monix-execution_2.12/3.0.0-M3/monix-execution_2.12-3.0.0-M3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/typelevel/cats-core_2.12/1.0.1/cats-core_2.12-1.0.1.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/io/monix/monix-reactive_2.12/3.0.0-M3/monix-reactive_2.12-3.0.0-M3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/typelevel/machinist_2.12/0.6.2/machinist_2.12-0.6.2.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/typelevel/cats-macros_2.12/1.0.1/cats-macros_2.12-1.0.1.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/io/monix/monix-tail_2.12/3.0.0-M3/monix-tail_2.12-3.0.0-M3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.4/scala-library-2.12.4.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-reflect/2.12.0/scala-reflect-2.12.0.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/jctools/jctools-core/2.1.1/jctools-core-2.1.1.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/io/monix/monix-eval_2.12/3.0.0-M3/monix-eval_2.12-3.0.0-M3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/io/monix/monix_2.12/3.0.0-M3/monix_2.12-3.0.0-M3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/typelevel/cats-effect_2.12/0.8/cats-effect_2.12-0.8.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/reactivestreams/reactive-streams/1.0.2/reactive-streams-1.0.2.jar",
        "quick": true
    }
]
```

and

```
gadgil@gadgil-Vostro-470:~/code/Superficial$ mill show freegroups.ammoniteReplClasspath
[1/1] show 
[
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/io/get-coursier/coursier_2.12/1.0.0-RC10/coursier_2.12-1.0.0-RC10.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/ammonite-repl_2.12.4/1.0.3/ammonite-repl_2.12.4-1.0.3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/ammonite-terminal_2.12/1.0.3/ammonite-terminal_2.12-1.0.3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/ammonite_2.12.4/1.0.3/ammonite_2.12.4-1.0.3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/io/get-coursier/coursier-cache_2.12/1.0.0-RC10/coursier-cache_2.12-1.0.0-RC10.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/ammonite-ops_2.12/1.0.3/ammonite-ops_2.12-1.0.3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/ammonite-compiler_2.12.4/1.0.3/ammonite-compiler_2.12.4-1.0.3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/ammonite-runtime_2.12/1.0.3/ammonite-runtime_2.12-1.0.3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/ammonite-util_2.12/1.0.3/ammonite-util_2.12-1.0.3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/upickle_2.12/0.4.4/upickle_2.12-0.4.4.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/github/javaparser/javaparser-core/3.2.5/javaparser-core-3.2.5.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/pprint_2.12/0.5.2/pprint_2.12-0.5.2.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/sourcecode_2.12/0.1.4/sourcecode_2.12-0.1.4.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/scalaj/scalaj-http_2.12/2.3.0/scalaj-http_2.12-2.3.0.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/fastparse-utils_2.12/1.0.0/fastparse-utils_2.12-1.0.0.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/scalaparse_2.12/1.0.0/scalaparse_2.12-1.0.0.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/jline/jline/2.14.3/jline-2.14.3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/scalaz/scalaz-core_2.12/7.2.13/scalaz-core_2.12-7.2.13.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/scalaz/scalaz-effect_2.12/7.2.13/scalaz-effect_2.12-7.2.13.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/scalaz/scalaz-concurrent_2.12/7.2.13/scalaz-concurrent_2.12-7.2.13.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-compiler/2.12.4/scala-compiler-2.12.4.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/fastparse_2.12/1.0.0/fastparse_2.12-1.0.0.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/spire-math/jawn-parser_2.12/0.10.3/jawn-parser_2.12-0.10.3.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/fansi_2.12/0.2.4/fansi_2.12-0.2.4.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.4/scala-library-2.12.4.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/geny_2.12/0.1.2/geny_2.12-0.1.2.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/lihaoyi/derive_2.12/0.4.4/derive_2.12-0.4.4.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-reflect/2.12.4/scala-reflect-2.12.4.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.12/1.0.6/scala-xml_2.12-1.0.6.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/org/javassist/javassist/3.21.0-GA/javassist-3.21.0-GA.jar",
        "quick": true
    },
    {
        "path": "/home/gadgil/.coursier/cache/v1/https/repo1.maven.org/maven2/com/github/scopt/scopt_2.12/3.5.0/scopt_2.12-3.5.0.jar",
        "quick": true
    }
]
```

**Comment by drhumlen** [comment]
Did you ever figure this one out? I'm experiencing `java.lang.IncompatibleClassChangeError: Implementing class` as well...

**Comment by siddhartha-gadgil** [comment]
No, and I confirmed earlier today that the bug persists in Mill 0.1.6 too.
It seems to be a clash of transitive dependencies - as my only dependency
was monix, if so it is some dependency of monix 3.0-M3 and Mill.

**Comment by ryan-williams** [comment]
I was seeing what I think was the same problem, and it seems to have been fixed somewhere in between `0.2.3` and `0.2.3-12-440700`
